charts/authelia/values.yaml

```diff
@@ -1305,6 +1305,7 @@ configMap:
     ##
     mysql:
       enabled: false
+      deploy: false
```

Comment:
What does this option do? It seems ineffectual.

Reply:
The `deploy` option toggles whether the `bitnami/mariadb` subchart is brought into play, and this condition is set in `Chart.yaml`. If the `mariadb` chart is activated, it will then consume the `mariadb` section of `values.yaml`.

Typically this condition would operate on `mysql.enabled` but I wanted to avoid breaking existing users of the chart, where people might want to enable mysql support but keep using their existing instance of mysql rather than have the chart deploy a new instance. So I created a new boolean for this.

Usually the `mysql` and `mariadb` sections would be merged into one section, taking the name of the subchart (so `mariadb`) but this would also break existing users so I chose not to refactor this at the moment. It's something we could talk about for a breaking release, though?

Reply:
Ah nice, didn't know abotu that interaction and didn't notice it! LGTM!

Reply:
Also as far as that change, I'm not entirely sure I care about a breaking change for 0.9.0, it's LONG been advertised it will be. See #140 and the target PR for more context. I think we can merge as is and decide later what we want to do. I like giving users the option to deploy it or use existing.

As far as MySQL/MariaDB users they're masochists who love things to break anyway. ;)